modules/core/src/main/scala/org/tessellation/http/routes/DebugTrustRoutes.scala

```diff
@@ -0,0 +1,57 @@
+package org.tessellation.http.routes
+
+import cats.effect.Async
+import cats.syntax.flatMap._
+import cats.syntax.functor._
+
+import org.tessellation.cli.method.ordinalTrustUpdateInterval
+import org.tessellation.domain.trust.storage.TrustStorage
+import org.tessellation.infrastructure.trust.storage.TrustStorage.TrustStore
+import org.tessellation.schema.SnapshotOrdinal
+import org.tessellation.schema.peer.PeerId
+import org.tessellation.schema.trust.{PublicTrust, SnapshotOrdinalPublicTrust}
+import org.tessellation.sdk.domain.gossip.Gossip
+import org.tessellation.security.hex.Hex
+
+import org.http4s._
+import org.http4s.circe.CirceEntityCodec.circeEntityEncoder
+import org.http4s.dsl.Http4sDsl
+import org.http4s.server.Router
+
+final case class DebugTrustRoutes[F[_]: Async](
+  gossip: Gossip[F],
+  trustStorage: TrustStorage[F]
+) extends Http4sDsl[F] {
+  private[routes] val prefixPath = "/debug/trust"
+
+  private val debug: HttpRoutes[F] = HttpRoutes.of[F] {
+    case GET -> Root / "latest" =>
+      val output = for {
+        trust <- trustStorage.getTrust
+        current <- trustStorage.getCurrentOrdinalTrust
+        next <- trustStorage.getNextOrdinalTrust
+      } yield TrustStore(trust, current, next)
+
+      output.flatMap(Ok(_))
+
+    case GET -> Root / "ordinalTrustGossip" =>
+      val publicTrust = SnapshotOrdinalPublicTrust(
+        SnapshotOrdinal(ordinalTrustUpdateInterval),
+        PublicTrust(
+          Map(
+            PeerId(
+              Hex(
+                "0174825ea2f952d7795475b690ffcc9960c486e7809fa9973e9db42cfc4f6360890d80a0e9a2363a5acf6c04cc188fa6405fae4bfdb9055b7361e0bbfe78941c"
+              )
+            ) -> 1.0
+          )
+        )
+      )
+
+      gossip.spread(publicTrust) >> Ok()
```

Comment:
I think we can have such functionality for a draft PR, but we shouldn't really have it for the final PR, even though it's the debug routes available only on non mainnet envs. It interacts with the actual internal state and processes rather then gives some insight into those.

Reply:
This endpoint has ben removed.